static/js/people.js

```diff
@@ -730,7 +730,7 @@ export function is_valid_email_for_compose(email) {
     if (!person) {
         return false;
     }
-    return active_user_dict.has(person.user_id);
+    return true;
```

Comment:
We should add a comment here noting that we need to allow deactivated users in compose so that one can attempt to reply to threads that contained them.

Reply:
Added the comment.